src/storage/src/hummock/compactor/mod.rs

```diff
@@ -801,7 +801,7 @@ async fn generate_splits(compact_task: &mut CompactTask, context: Arc<Context>)
         .sum::<u64>();
 
     let sstable_size = (context.options.sstable_size_mb as u64) << 20;
-    if compaction_size > sstable_size {
+    if compaction_size > sstable_size * 2 {
```

Comment:
Can we use `sstable_size * max_sub_compaction` to ensure the min_size of splits larger than single `sstable_size`